### DISCUSSION TITLE
I'm not receiving daily newsletters anymore. Anyone with the same issue?

### DISCUSSION THREAD

**Discussion by makumi10** [community/community#68774] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

GitHub used to email daily trending news and recommended repos based on my interests, but recently stopped despite my subscriptions being active. Unsure why I no longer receive these expected digests even though my profile settings still show enrolled. Hoping to resolve soon so I can again get GitHub's daily newsletter.

**Comment by makumi10** [comment, marked as answer]
Check Your GitHub Notification Settings:

Go to your GitHub account settings by clicking on your profile picture in the top right corner and selecting "Settings."
In the left sidebar, click on "Notifications."
Ensure that your email notification preferences are correctly configured. Specifically, make sure that "Participating" and "@mentions" are set to "Email" and "On GitHub.
Sometimes, GitHub notification emails can end up in your spam or junk folder. Check these folders to see if the emails are being filtered there.

**Comment by queenofcorgis** [comment]
Hi USERNAME,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was closed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
